cybersixgill_actionable_alerts/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,58 @@
+name: cybersixgill_actionable_alerts
+files:
+- name: cybersixgill_actionable_alerts.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+      - name: cl_id
+        required: true
+        description: The Client Id given by Cybersixgill
+        enabled:  true
+        value:
+          type: string
+          example:  clientid
+          display_default:  null
+      - name: cl_secret
+        required: true
+        description:  The Client Secret given by Cybersixgill
+        enabled:  true
+        value:
+          type: string
+          example:  client00000secret
+          display_default:  null
+      - name: alerts_limit
+        required: false
+        description:  The number of alerts to fetch on a single request default is 50
+        enabled:  false
+        value:
+          type: string
+          example:  "50"
+          display_default:  null
+      - name: threat_type
+        required: false
+        description:  Predefined types of threats alerts you would like to see like fraud, malware
```

Comment:
Is there a set list of valid types of threats? If so, we could convert this into an `enum` similar to https://github.com/DataDog/integrations-core/blob/1a38012c974eca7d3714374510ddad62a438f00c/teradata/assets/configuration/spec.yaml#L98-L105

Reply:
@shahul-loginsoft Please see the above